planner/cascades/testdata/transformation_rules_suite_in.json

```diff
@@ -25,7 +25,12 @@
     "name": "TestTopNRules",
     "cases": [
       "select b from t order by a limit 2",
-      "select a+b from t order by a limit 1 offset 2"
+      "select a+b from t order by a limit 1 offset 2",
+      "select c from t order by t.a limit 1",
+      "select c from t order by t.a + t.b limit 1",
+      "select 1 as a from dual order by a limit 1",
+      "select * from t t1 where t1.a in (select t2.a as a from t t2 where t2.b > t1.b order by t1.b limit 1)",
+      "select * from t t1 where t1.a in (select a from (select t2.a as a, t1.b as b from t t2 where t2.b > t1.b) x order by b limit 1)" 
```

Comment:
Use something like `select a, b, c` instead of `select * ` to reduce the length of schema.